webpack.config.performance.test.js

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// This file uses CommonJS.
+/* eslint-disable import/no-commonjs */
+'use strict';
+
+const path = require('path');
+
+const glob = require('glob');
+const {merge} = require('webpack-merge');
+
+const base = require('./webpack.config.base');
+
+const WEBSERVER_PORT = process.env.WEBSERVER_PORT ?? 9001;
+
+module.exports = merge(base, {
+    entry: {
+        e2e: glob.sync('./e2e/performance/**/*.test.js'),
```

Comment:
This covers a folder already covered by `webpack.config.test.js`, so that file would also run everything in here. Should we exclude these tests from that file?

Reply:
Ahh you're right, nevermind :)